### ISSUE TITLE
Zypper info output in quiet mode does not make sense

### ISSUE THREAD

**Issue by Fdall** [openSUSE/zypper#287]
Until at least zypper 1.12.23 I was using in a script the following command:
zypper --quiet info <package/pattern name>
Which gave the same output as its non-quiet equivalent.

On a more recent version the output only the header is displayed:

````
agent:~ # zypper --quiet info vim


Information for package vim:
----------------------------
````
In the doc we have: "**Suppress normal output, print only error messages**."
Which was not the old behaviour since the normal output was kept, and is not the new one since the header only is kept.

Should I remove the --quiet since it should not output anything at all? or is the actual output not behaving as expected?

**Comment by mlandres** [comment]
No, I'd expect it to behave like `search`. Repo refresh noise etc. should be suppressed but the requested information should be printed. We'll fix it.

**Comment by mlandres** [comment]
Fixed for zypper-1.14.30
